travel_guide.rst

```diff
@@ -51,9 +51,11 @@ there may not be much public street parking near the University.
 Driving
 -------
 
-The University is accessible from the I-5, at the 45th St exit. There is some
-parking on campus [to be confirmed]. Prices / availablity TBD.
+The University is accessible from the I-5, at the 45th St exit. Follow [directions to central campus](https://transportation.uw.edu/getting-here/drive?ref=#directions-to-central-campus). The conference is in Kane Hall on Red Square. There is parking both on campus and on the street near the campus. Parking on campus is the best option Saturday. Both are free on Sunday.
```

Comment:
Hi - can you do a quick fix here - this is the wrong format for links in RST, you need:
```
`Link text <https://domain.invalid/>`_
```